### DISCUSSION TITLE
Update: EOL Trovis-Regler (5571, 75, 76, 79)

### DISCUSSION THREAD

**Discussion by Tom-Bom-badil** [Tom-Bom-badil/samson_trovis_557x#31] [General (de/en)]
Kurz am Rande: Nachdem ich mich heute beim Überarbeiten des Wiki's gewundert habe, warum einige ältere Links ins Leere laufen, habe ich die aktuelle Samson-Seite mal etwas genauer unter die Lupe genommen. Dabei ist mir folgendes aufgefallen:

[Aufgeführt / beworben](https://www.samsongroup.com/en/products/automation-systems/)  werden dort nur noch folgende Regler:
- 5573, 5573-1
- 5578, 5578-E

Folgende Regler tauchen NICHT mehr auf:
- 5571 (bereits als 'discontinued' gekennzeichnet)
- 5575 (bereits als 'discontinued' gekennzeichnet)
- 5576 
- 5579

Ich hatte mich jüngst schon gewundert, warum es für meine 5576 und 5579 keine Firmwareaktualisierungen auf 3.00 gab - bei 2.63 war Schluss. Die Firmwares und Dokumente für diese Regler konnte ich heute nur noch über die Suche finden, die Links zu den eigentlichen Produktseiten laufen ins Leere.

Es scheint also so zu sein, dass es sich bei allen 4 Reglern um Auslaufmodelle handelt. Wer sich also noch die Handbücher, Datenblätter etc zu den aktuellen Firmwareständen als pdf wegspeichern oder seinen Regler auf den aktuellen Firmwarestand bringen will, sollte dies vermutlich bald tun - hier kann ich solche Dinge natürlich aus Copyrightgründen nicht einstellen.

/tom

_p.s./edit: Interessant ist in diesem Zusammenhang die Kompatibilität der Sockel. Aus meinem eigenen Umbau weiß ich, dass die 5575 definitiv einen anderen Sockel hat als die 5576/5579 (deren beiden Sockel sind gleich). Zumindest laut Verkabelungsplan und mechanischem Aufbau laut Handbüchern scheint im Falle eines Reglerdefekts folgender Austausch möglich zu sein: 5576/5579 --> 5578/5578-E, 5575 --> 5573/5573-1._

**Comment by yufaxt** [comment]
Auf der Webseite steht die Version 3.00 nur beim 5578-E und nicht beim 5578. Bei mir ist übrigens wegen der zwei 0-10V Regler nur der 5578-E relevant, weil es nach dem Schaltplan vom „normalen“ 5578 nur einen 0-10V Ausgang gibt. So verstehe ich das zumindest.

5578-1113 und 5578-1114 sind anscheinend 5578-E und wären daher als Ersatz für meinen 5576 im Fälle eines Defekts relevant.

**Comment by Tom-Bom-badil** [comment]
Stimmt, gemäß [diesem Dokument](https://www.samsongroup.com/document/t55000de.pdf) haben nur die 5576, 5579 und 5578-E mehr als nur einen 0-10V Ausgang. Danke für den Hinweis zur 3er Firmware bei der 5578, ist mir tatsächlich nicht aufgefallen! :)

/tom

**Comment by Tom-Bom-badil** [comment]
> Zumindest laut Verkabelungsplan und mechanischem Aufbau laut Handbüchern scheint im Falle eines Reglerdefekts folgender Austausch möglich zu sein: 5576/5579 --> 5578/5578-E, 5575 --> 5573/5573-1.

Nachtrag: Der direkte Austausch ohne Sockeltausch ist **NICHT** möglich. Zumindest lässt sich keine 5579 durch eine neuere 5578 ersetzen. Der Grund ist die komplette Stiftleiste auf der 230V-Seite. Diese sitzt bei der 5578 ca. 2.5 mm weiter innen - und somit passt die Stiftleiste des Reglers meachanisch nicht in die Löcher der Buchsenleiste des alten Sockels:

![Trovis_Sockel](https://github.com/user-attachments/assets/707087b6-08fa-4401-a037-dc58a2e81d9b)
(nur Beispielbild zur Verdeutlichung - der echte 5579-Sockel sieht links natürlich anders aus)

Hab nochmal die Anschlusspläne geprüft - rein elektrisch müsste es gehen, nur mechanisch ist der Aufbau halt ein anderer. Die Beschaltung ist bis auf die eher selten genutzten Buchsen 19-28 (0-10V, Samson Raumregler, Pumpendrehzahl usw) identisch. Schade, hätte ja klappen können. :(

/tom

_p.s. Ich probier demnächst nochmal die 5579 im 5576-Sockel ..._

**Comment by Nesselbrand67** [comment]
Ich habe heute auf der Samson Webseite nachgeschaut (die die oben verlinkt ist) und nun ist auch die 5578 nicht mehr aufgeführt.

Aktuelle sind nur noch 5578-E, 5573 und 5573-1.

Erklärt auch, warum die 5578 nicht die V3.00 FW bekommen hat.

Nachtrag: Suche ich nach 5578, so erhalte ich einen Eintrag für die Bedienungsanleitung mit dem Hinweis "abgekündigtes Produkt"
